pkg/otlp/model/translator/dimensions.go

```diff
@@ -30,6 +30,7 @@ const (
 
 // Dimensions of a metric that identify a timeseries uniquely.
 // This is similar to the concept of 'context' in DogStatsD/check metrics.
+// NOTE: Keep this in sync with the TestDimensions struct.
```

Comment:
Could a test validate this? Otherwise how do you expect this to happen?

Reply:
I would expect that we change this very rarely and we just manually check it. I may be able to do some fancy reflection to validate this (iterate over the fields on each struct and assert they are equivalent), would that make sense to you?

Reply:
I think it's worth doing that only if them getting out of sync can cause tests to potentially miss bugs.

Reply:
Added a unit test for this